### ISSUE TITLE
On the stars selected control, narrator is announcing incorrect number limit information i.e min zero and max zero.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#130]
User experience:
On any control If, the Narrator is announcing incorrect number limit information i.e min zero and max zero, then the user who depends upon Narrator for navigation might get affected hence failed to get proper information of the control.

Note: User credentials should NOT be included in the bug.

Pre-requisites:

1. Open developer toolbar via pressing F12 key.
2. Open Narrator via pressing ctrl+window+enter key.
 

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Testimonial and select it.
8.Navigate to the "Testimonial" dialog and select edit button.
9. Navigate with the Narrator on the "Testimonial" dialog box and reach stars selected control.
10. Verify, whether the Narrator is announcing proper information of the number limit on the control or not.

Actual Result:

On the stars selected control, narrator is announcing incorrect number limit information i.e min zero and max zero. 

Expected Result:

On the stars selected control, narrator should announce correct number limit information i.e min  zero and max 10. So, that the Screen Reader user does able to get proper information of the number limit.

Note:

1. Similar issue exist for all stars (max 10) control.
2. This issue does not exist with Chrome +JAWS+NVDA. JAWS and NVDA does not announce the number limit information.
 

MAS Reference:

MAS 4.2.1 - Object Information

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

![image](https://user-images.githubusercontent.com/2320302/79021554-b1366480-7b30-11ea-9c42-af6522f3bab7.png)
